### DISCUSSION TITLE
The page is out of order

### DISCUSSION THREAD

**Discussion by ghost** [community/community#12271] [Other Feature Feedback, Questions, & Ideas]
Disordered location:  https://github.com/hai2007/clunch/network
![屏幕截图 2022-02-28 171743](https://user-images.githubusercontent.com/73682414/156287022-a3ad9109-7f5d-472b-99a6-f683ef51b3b1.png)

The content is not fully displayed. It was good before. It may have changed something wrong.

**Comment by yelloxing** [comment]
This is a bug.
